browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3792,6 +3792,12 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_SETTINGS_ENABLE_NFT_DISCOVERY_DESC" desc="The description for enabling NFT discovery">
       Automatically add NFTs you own to the Wallet using third party APIs. <ph name="BEGIN_LINK_1">&lt;LINK_1&gt;</ph>Learn more.<ph name="END_LINK_1">&lt;/LINK_1&gt;</ph>
     </message>
+    <message name="IDS_SETTINGS_ENABLE_BRAVE_WALLET_IN_PRIVATE_WINDOWS_LABEL" desc="The label for enabling NFT discovery">
+      Enable Brave Wallet in Private Windows.
+    </message>
+    <message name="IDS_SETTINGS_ENABLE_BRAVE_WALLET_IN_PRIVATE_WINDOWS_DESC" desc="The description for enabling NFT discovery">
+      Enable Brave Wallet in Private Windows.
```

Comment:
I'll confirm if we want this description and if so what it should say.

Reply:
We need to display a warning `Warning: Enabling this exposes you to the possibility of sites using fingerprinting data to link a private window session to a non-private window session.` as mentioned in https://github.com/brave/reviews/issues/1145#issuecomment-1915824015.